native/packages/react-native-bpk-component-switch/src/BpkSwitch.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Platform,
+  Switch,
+} from 'react-native';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withTheme } from 'react-native-bpk-theming';
+
+const tokens = Platform.select({
+  ios: () => require('bpk-tokens/tokens/ios/base.react.native.common.js'), // eslint-disable-line global-require
+  android: () => require('bpk-tokens/tokens/android/base.react.native.common.js'), // eslint-disable-line global-require
+})();
+
+const getColors = (theme, value) => {
+  const primaryColor = theme ? theme[THEMING_ATTRIBUTE] : tokens.colorBlue500;
+  const secondaryColor = tokens.colorGray100;
+
+  // The color props mean different things based on the platform.
+  const colors = Platform.select({
+    ios: {
+      tintColor: secondaryColor, // Border around the switch when off.
+      thumbTintColor: null, // Must be unset or the the handles loses its drop shadow.
```

Comment:
https://github.com/facebook/react-native/issues/16764

Reply:
Can we include a reference to this issue in the source too?

Reply:
Sure